src/pages/api/notifications.ts

```diff
@@ -0,0 +1,69 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import { HTTP_STATUS } from "../../constants/http";
+import { services } from "../../services/ServiceLocatorServer";
+import { Notification } from "../../services/NotificationsStore";
+
+async function fetchNotifications() {
+  return await services().getNotificationsStore().getNotifications();
+}
+
+function isNotification(o: any): o is Notification {
+  return !!(o.content && o.id && o.type);
+}
+
+function invalidArgumentError() {
+  return { err: HTTP_STATUS.INVALID_REQUEST };
+}
+
+function asNotification(o: object): Notification {
+  if (isNotification(o)) return o as Notification;
+  else throw invalidArgumentError();
+}
+
+function asNotifications(body: any): Notification[] {
+  const { notifications } = body;
+  if (Array.isArray(notifications)) {
+    throw invalidArgumentError();
+  }
```

Comment:
Should this be `if (!Array.isArray(notifications)) {` ?

Reply:
Code removed :-)